methods_and_recursion/mad_libs.rb

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
+
+def ask_question(question)
+  puts "Give me #{question}:"
+  gets.chomp
+end
+
+adj = ask_question('an adjective')
+adj_2 = ask_question('another adjective')
+noun = ask_question('a noun')
+noun_2 = ask_question('another noun')
+plural_noun = ask_question('a plural noun')
+game = ask_question('a game')
+plural_noun_2 = ask_question('abother plural noun')
+verb_ing = ask_question("a verb ending in 'ing'")
+verb_ing_2 = ask_question("another verb ending in 'ing'")
+plural_noun_3 = ask_question('another plural noun')
+verb_ing_3 = ask_question("another verb ending in 'ing'")
+noun_3 = ask_question('another noun')
+plant = ask_question('a plant')
+part_of_the_body = ask_question('a part of the body')
+place = ask_question('a place')
+verb_ing_4 = ask_question("another verb ending in 'ing'")
+adj_3 = ask_question('another adjective')
+number = ask_question('a number')
+plural_noun_4 = ask_question('another plural noun')
+
+# rubocop:disable Style/StringLiterals
+puts """
```

Comment:
Just for something new... try `puts <<-END_OF_MADLIB`.  Google for Ruby here document for more info.